packages/terra-tabs/src/Tabs.jsx

```diff
@@ -169,35 +178,44 @@ class Tabs extends React.Component {
       React.cloneElement(contentItem, { isLabelHidden: isIconOnly || this.state.isLabelTruncated })
     ));
 
-    const collapsibleTabs = (
-      <CollapsibleTabs
-        activeKey={activeKey || this.state.activeKey}
-        activeIndex={this.getActiveTabIndex()}
-        onChange={this.handleOnChange}
-        onTruncationChange={this.handleTruncationChange}
-        variant={variant}
-      >
-        {clonedPanes}
-      </CollapsibleTabs>
-    );
+    const header = (() => {
+      const collapsibleTabs = (
+        <CollapsibleTabs
+          activeKey={activeKey || this.state.activeKey}
+          activeIndex={this.getActiveTabIndex()}
+          onChange={this.handleOnChange}
+          onTruncationChange={this.handleTruncationChange}
+          variant={variant}
+        >
+          {clonedPanes}
+        </CollapsibleTabs>
+      );
+
+      if ([DependentViewport.PARENT, DependentViewport.WINDOW].includes(responsiveTo)) {
+        const collapsedTabs = (
+          <CollapsedTabs activeKey={activeKey || this.state.activeKey} onTruncationChange={this.handleTruncationChange}>
+            {clonedPanes}
+          </CollapsedTabs>
+        );
+
+        return (
+          <ResponsiveElement
+            tiny={collapsedTabs}
+            small={collapsibleTabs}
+            responsiveTo={responsiveTo}
```

Comment:
Will these changes be valid after https://github.com/cerner/terra-framework/pull/972 is merged?

Reply:
I don't think so; the core logic should probably remain as-is. I think what gets rendered now should be based off of both `breakpoint` value and `responsiveTo` value - something like —

```
if ((current breakpoint is "tiny") AND (responsiveTo is either "parent" or "window")) {
    <---- show collapsed tabs ---->
 } else {
    <---- show collapsible tabs ---->
}
```